lib/segment/src/data_types/index.rs

```diff
@@ -198,6 +202,129 @@ pub struct TextIndexParams {
     // todo(phrase_match): remove skip
     #[serde(skip)]
     pub phrase_matching: Option<bool>,
+
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub stopwords: Option<StopwordsInterface>,
+}
+
+#[derive(Debug, Serialize, Deserialize, JsonSchema, Clone, PartialEq, Hash, Eq)]
+#[serde(untagged)]
+pub enum StopwordsInterface {
+    Language(Language),
+    Set(StopwordsSet),
+}
+
+impl StopwordsInterface {
+    #[cfg(feature = "testing")]
```

Comment:
What exactly is the feature "testing" for? What is the difference compared to `#[cfg(test)]`?

Reply:
It was introduced to use across crates